tools/accuracy_checker/tests/test_presenter.py

```diff
@@ -657,37 +657,6 @@ def test_vector_presenter_with_vector_data_with_vector_scale(self, mocker):
         ]
         mock_write_scalar_res.assert_has_calls(calls)
 
-    def test_vector_presenter_with_vector_data_with_dict_ref(self, mocker):
```

Comment:
why do you remove test for testing reference represented as dictionary in the config?

Reply:
I return this test, but I removed scaling for vector metric at all. Now scale using only for present metric. So when we calculate `mean` of vector metric we don't multiply values by scale.